src/IKEA/Tradfri/Helper/Online.php

```diff
@@ -14,33 +14,47 @@ class Online
     /**
      * Check online state.
      *
-     * @param string $ip
+     * @param string $ipAddress
      *
      * @return bool
      */
-    public static function isOnline(string $ip): bool
+    public static function isOnline(string $ipAddress): bool
     {
+        $online = true;
+        $regex = '/(\ \d+% packet loss)/';
```

Comment:
This regex doesn't work on a mac. I would suggest: (\ [\d\.]+% packet loss).

Example output of ping command:

PING xxxxxxxxx(192.168.1.15): 56 data bytes
64 bytes from xxxxxxxxx: icmp_seq=0 ttl=128 time=0.197 ms

--- xxxxxxxxx ping statistics ---
1 packets transmitted, 1 packets received, 0.0% packet loss
round-trip min/avg/max/stddev = 0.197/0.197/0.197/0.000 ms

Reply:
Hi, I changed it to your suggestion thx!